README.md

```diff
@@ -0,0 +1,15 @@
+# ROOMs app for Rebuilding Together Peninsula
+
+*Copyright 2010 Luke Stone*
+
+This is an online ordering system for construction captains for the
+annual Rebuilding Day.
+
+### Developer Setup
+1. [Install the google sdk](https://cloud.google.com/sdk/docs/)
+1. `git clone git@github.com:babybunny/rebuildingtogethercaptain.git`
+1. `cd rebuildingtogethercaptain`
+1. `virtualenv virtualenv  # this directory name is .gitignored`
+1. `./virtualenv/bin/pip install -r requirements.txt`
+1. `./virtualenv/bin/python run_tests.py /path/to/google-sdk-install`  # run the tests
```

Comment:
TODO: Instead of the argument here, there should be a step beforehand (probably after installing the sdk) to make sure PYTHONPATH is set up appropriately with a test to launch interpreter and try:
>>> import dev_appserver.fix_sys_path